### DISCUSSION TITLE
Employee Expense Reports

### DISCUSSION THREAD

**Discussion by snunez1** [ledgersmb/LedgerSMB#8266] [Ideas]
I'm in need of an accounting system for a professional services startup.  We're in technology, and so far LedgerSMB seems to fit the bill in that it's simple and we understand PostgreSQL and PERL. Except for the lack of employee travel expense reporting.

Payroll we can outsource to the accountant because that follows a set formula, but travel expenses vary widely, and we need a way to automate this.

We can't be the first ones to need this in LedgerSMB, so I want to ask here:

- Has this been done before and is missing from the documentation?
- Is it a work-in-progress with anyone?
- Are there any free (ideally open-source) expense management solutions that can be integrated?
- Any work-arounds within LedgerSMB?

Really hoping LedgerSMB turns out to be 'the One' and we can stop looking and get to work.  If we can get over the travel expense reporting, we're golden.

**Comment by neilt** [comment]
Thank you for considering LSMB.

* Nope not done before, except with the workaround mentioned in my reply in #8267 
* Nope not in progress, at least that I am aware of
* Any integrations would require code support
* For a workaround see my response in #8267

This functionality could be added with sponsorship, or code contributions.

**Comment by ehuelsmann** [comment]
In order to be able to pick this up, or to be able to estimate how difficult the request is, could you add an example expense report and document what the workflow with it would be? Like: which actors do you expect (Expense-reimbursement requestor, Expense approver, cash payment handler/accountant, others)? Which steps do you expect each to take? Which aggregated reports do you need? Do you need a list of work-in-progress and maybe expensed reports over a certain period?

The more you have, the easier it will be for me to understand what I'm up against here. (Although I'd be very happy for anybody else to pick this up before I get to it!)

**Comment by snunez1** [comment]
The [expenses workflow in flectra](https://doc.flectrahq.com/3.0/applications/finance/expenses.html) seems to tick all the boxes.
